format/spec.md

```diff
@@ -245,19 +245,19 @@ For example, a file may be written with schema `1: a int, 2: b string, 3: c doub
 
 Tables may also define a property `schema.name-mapping.default` with a JSON name mapping containing a list of field mapping objects. These mappings provide fallback field ids to be used when a data file does not contain field id information. Each object should contain
 
-* `names`: A required list of 0 or more names for a field. 
+* `names`: A required list of 0 or more names for a field.
 * `field-id`: An optional Iceberg field ID used when a field's name is present in `names`
 * `fields`: An optional list of field mappings for child field of structs, maps, and lists.
 
 Field mapping fields are constrained by the following rules:
 
-* A name may contain `.` but this refers to a literal name, not a nested field. For example, `a.b` refers to a field named `a.b`, not child field `b` of field `a`. 
-* Each child field should be defined with their own field mapping under `fields`. 
```

Comment:
If possible, can we undo all the changes which are space removals at the end of the line? It'll be easier to review the PR that way

Reply:
Done